autoload/characterize.vim

```diff
@@ -3281,8 +3626,6 @@ let s:d[0x09E0]='BENGALI LETTER VOCALIC RR'
 let s:d[0x09E1]='BENGALI LETTER VOCALIC LL'
 let s:d[0x09E2]='BENGALI VOWEL SIGN VOCALIC L'
 let s:d[0x09E3]='BENGALI VOWEL SIGN VOCALIC LL'
-let s:d[0x09E4]='<reserved>'
-let s:d[0x09E5]='<reserved>'
```

Comment:
I also think it's better not to lose all of the `<reserved>` and `<not a character>` entries.